src/engine/Unit_engine.ml

```diff
@@ -701,7 +702,8 @@ let tainting_test lang rules_file file =
     |> List.concat_map (fun rule ->
            let xtarget =
              {
-               Xtarget.file;
+               Xtarget.source = File file;
+               file;
                xlang = Xlang.L (lang, []);
                lazy_content = lazy (UFile.read_file file);
```

Comment:
same here. I would write:
```
let xtarget : Xtarget.t = {
  source = ...; ...
}

Reply:
Done in 135dff8da0424f9a79b39d7a72e2c7c10936e403